readme.md

```diff
@@ -90,4 +90,4 @@ Run `/audioplayer name` while holding a music disc, goat horn or head with custo
 - [Simple Voice Chat](https://github.com/henkelmax/simple-voice-chat)
 - [Admiral](https://github.com/henkelmax/admiral)
 
-*Note that the files you upload to Filebin are publicly available if the upload link is disclosed!*
\ No newline at end of file
+*Note that the files you upload to Filebin are publicly available if the upload link is disclosed!*
```

Comment:
What did you change here?

Reply:
Nothing, it just appears like I changed something. I don't know why.